modules/apps/commerce/commerce-order-content-web/src/main/java/com/liferay/commerce/order/content/web/internal/frontend/data/set/provider/PreviewCommerceOrderItemFDSDataProvider.java

```diff
@@ -134,6 +134,28 @@ public List<PreviewOrderItem> getItems(
 				CommerceOrderItemPrice commerceOrderItemPrice =
 					commerceOrderImporterItem.getCommerceOrderItemPrice();
 
+				if ((commerceOrderItemPrice != null) &&
+					commerceOrderItemPrice.isPriceOnApplication()) {
+
+					return new PreviewOrderItem(
+						externalReferenceCode,
```

Comment:
reviewing the wireframe we found out that the columns promoprice, discounts and total should render the dash '-' character

Reply:
Fixed